src/screens/OtherProfile/index.js

```diff
@@ -159,20 +92,14 @@ const OtherProfile = () => {
   const [loading, setLoading] = React.useState(false);
   const [initLoading, setInitLoading] = React.useState(true);
   const [isAnonimity, setIsAnonimity] = React.useState(false);
-  const {saveChatFromOtherProfile, savePendingChatFromOtherProfile} = useSaveAnonChatHook();
   const {params} = route;
   const {feeds} = otherProfileFeeds;
   const [loadingGenerateAnon, setLoadingGenerateAnon] = React.useState(false);
-  const [loadingSendDM, setLoadingSendDM] = React.useState(false);
   const [anonProfile, setAnonProfile] = React.useState();
-  const [dmChat, setDMChat] = React.useState();
   const {mappingColorFeed} = useCoreFeed();
   const isSignedMessageEnabled = dataMain.isSignedMessageEnabled ?? true;
   const isAnonimityEnabled = dataMain.isAnonMessageEnabled && isSignedMessageEnabled;
   const {createSignChat} = useCreateChat();
-  React.useEffect(() => {
-    setDMChat('');
-  }, [isSignedMessageEnabled]);
 
   const generateAnonProfile = async () => {
     setLoadingGenerateAnon(true);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The state variable `loadingSendDM` and `dmChat` have been removed. If these variables were used for sending direct messages, ensure that their removal does not affect the functionality of the application.
```diff
- const [loadingSendDM, setLoadingSendDM] = React.useState(false);
- const [dmChat, setDMChat] = React.useState();
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->